### PR TITLE
[CHORE] update ember version

### PR DESCRIPTION
**Pull request by LiquidSean**
Update ember version to 2.17 and remove ember-cli-shims from package.json

### CODE REVIEW

**Comment by tylerbrazier** [comment]
looks like ember 3.0 is out. do you think we may as well just get the latest?

**Comment by tylerbrazier** [comment]
Also, are we sure we need all those files in `.idea/`?

**Comment by LiquidSean** [comment]
@tylerbrazier I will look at updating to ember 3.0. Also those .idea/ files that is my bad. Did not notice I committed all those.

**Comment by tylerbrazier** [comment]
please remove .idea/workspace.xml and .DS_Store

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-{{welcome-page}}
+{{count-up
+  startVal=0
+  endVal=100
+  duration=1.5
+  decimals=2
+  useEasing=true}}
 {{!-- Feel free to remove this! --}}
 
 {{outlet}}
```
Comment (tylerbrazier) [inline tests/dummy/app/templates/application.hbs]: please add newline

**Comment by tylerbrazier** [comment]
LGTM 👍

**Pull request merged**